cmd/gnodev/precompile.go

```diff
@@ -13,19 +13,21 @@ import (
 )
 
 type precompileOptions struct {
-	Verbose     bool   `flag:"verbose" help:"verbose"`
-	SkipFmt     bool   `flag:"skip-fmt" help:"do not check syntax of generated .go files"`
-	GoBinary    string `flag:"go-binary" help:"go binary to use for building"`
-	GofmtBinary string `flag:"go-binary" help:"gofmt binary to use for syntax checking"`
-	Output      string `flag:"output" help:"output directory"`
+	Verbose     bool            `flag:"verbose" help:"verbose"`
+	SkipFmt     bool            `flag:"skip-fmt" help:"do not check syntax of generated .go files"`
+	GoBinary    string          `flag:"go-binary" help:"go binary to use for building"`
+	GofmtBinary string          `flag:"go-binary" help:"gofmt binary to use for syntax checking"`
+	Output      string          `flag:"output" help:"output directory"`
+	skipPkgs    map[string]bool `flag:"-"`
```

Comment:
I don't think it's a good idea to modify `precompileOptions` to have a field that you modify down the line.
I believe the idea behind flag options is that they are modified once (upon initialization), and not continuously.

You continuously modify `skipPkgs` in a loop, instead of using a local variable or even a different pipeline for precompiling these files and skipping some

Reply:
Resolving this convo as you've rewritten the implementation